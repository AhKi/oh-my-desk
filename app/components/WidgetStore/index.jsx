import React from 'react';
import './WidgetStore.scss';

class WidgetStore extends React.Component {
	render() {
		return (
			<div className="WidgetStore">
				<div className="WidgetStore__content">
					<h4>{'"'}Store{'"'}기능은 <br />준비 중 입니다.</h4>
				</div>
			</div>
		);
	}
}

export default WidgetStore;
