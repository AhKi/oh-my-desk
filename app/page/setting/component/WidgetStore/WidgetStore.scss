.WidgetStore {
  display: flex;
  padding: 20px;
  width: 100%;
  height: 100%;

  align-content: center;
  justify-content: center;
  align-items: center;
}

.WidgetStore__content {
  border: 1px solid #e1e1e1;
  border-radius: 4px;
  background-color: #f5f5f5;
  display: flex;
  height: 300px;
  width: 80%;
  line-height: 1.3em;

  align-items: center;
  justify-content: center;
}
