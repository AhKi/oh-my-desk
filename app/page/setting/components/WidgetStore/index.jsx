import React from 'react';
import './WidgetStore.scss';

class WidgetStore extends React.Component {
  render() {
    return (
      <div className="WidgetStore">
        <div className="WidgetStore__content">
          <h4>
            {'"'}
            Store
            {'"'}
            function is
            <br />
            Comming soon.
          </h4>
        </div>
      </div>
    );
  }
}

export default WidgetStore;
