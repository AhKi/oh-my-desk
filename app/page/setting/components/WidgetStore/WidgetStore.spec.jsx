import React from 'react';
import { shallow } from 'enzyme';

import WidgetStore from '.';

describe('<WidgetStore />', () => {
  it('should match to snapshot when render default', () => {
    const wrapper = shallow(<WidgetStore />);

    expect(wrapper).toMatchSnapshot();
  });
});
