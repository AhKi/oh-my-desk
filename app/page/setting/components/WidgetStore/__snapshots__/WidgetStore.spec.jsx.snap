// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WidgetStore /> should match to snapshot when render default 1`] = `
<div
  className="WidgetStore"
>
  <div
    className="WidgetStore__content"
  >
    <h4>
      "
      Store
      "
       function is 
      <br />
      Comming soon.
    </h4>
  </div>
</div>
`;
